rotkehlchen/tests/unit/decoders/test_optimism_gateway.py

```diff
@@ -0,0 +1,328 @@
+            event_identifier=evmhash,
+            sequence_index=0,
+            timestamp=TimestampMS(1674128075000),
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal('0.01405791999714114')),
+            location_label=user_address,
+            notes='Burned 0.01405791999714114 ETH for gas',
+            counterparty=CPT_GAS,
+        ), HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=121,
+            timestamp=TimestampMS(1674128075000),
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=Asset('eip155:1/erc20:0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48'),
+            balance=Balance(amount=FVal('2718.857536')),
+            location_label=user_address,
+            notes=f'Bridge 2718.857536 USDC from optimism address {user_address} to ethereum address {user_address} via optimism gateway',  # noqa: E501
+            counterparty=CPT_OPTIMISM,
+        ),
+    ]
+
+
+@pytest.mark.vcr
+@pytest.mark.parametrize('ethereum_accounts', [['0xE232E72983E329757F02292322296f5B96dAfC8F']])
+def test_claim_eth_on_ethereum(database, ethereum_inquirer, ethereum_accounts):
+    """Data is taken from
+    https://etherscan.io/tx/0x785df1d7f2cb08ba956792393a2d947f1c18b39bd924e238811e4a14da10b0c4
+    """
+    evmhash = deserialize_evm_tx_hash('0x785df1d7f2cb08ba956792393a2d947f1c18b39bd924e238811e4a14da10b0c4')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=evmhash,
+    )
+    user_address = ethereum_accounts[0]
+    assert events == [
+        HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=0,
+            timestamp=TimestampMS(1674127139000),
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal('0.012214330132870492')),
+            location_label=user_address,
+            notes='Burned 0.012214330132870492 ETH for gas',
+            counterparty=CPT_GAS,
+        ), HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=1,
+            timestamp=TimestampMS(1674127139000),
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal('0.435796826762301485')),
+            location_label=user_address,
+            notes=f'Bridge 0.435796826762301485 ETH from optimism address {user_address} to ethereum address {user_address} via optimism gateway',  # noqa: E501
```

Comment:
```suggestion
            notes=f'Bridge 0.435796826762301485 ETH from optimism address {user_address} to ethereum address {user_address} via optimism bridge',  # noqa: E501
```

Just a reminder to make sure it's also changed here